exe/etcd-env

```diff
@@ -5,14 +5,17 @@ require 'json'
 require 'net/http'
 
 def usage
-  puts "Usage: #{$PROGRAM_NAME} <etcd key> <command>"
+  puts "Usage: #{$PROGRAM_NAME} <etcd key> [ <etcd key> ... ] -- <command>"
 end
 
-def env_vars(key)
-  response = fetch "http://#{etcd_host}:#{etcd_port}/v2/keys#{key}"
-  (JSON.parse(response)['node']['nodes'] || [])
-    .map { |var| [var['key'].split('/').last, var['value']] }
-    .to_h
+def env_vars(keys)
+  keys.map! do |key|
```

Comment:
I like when you say : do something like this, and it's actually exactly this, with just 2 parenthesis missing.